cli/src/main/kotlin/commands/ReporterCommand.kt

```diff
@@ -237,8 +237,9 @@ class ReporterCommand : CliktCommand(
             println("Successfully created the '${reporter.reporterName}' report at '$file' in ${duration.inSeconds}s.")
         }
 
+        println("Successfully created ${reportDurationMap.size} of ${reports.size} report(s).")
```

Comment:
I think there should be a different message if no report was created successfully.

Reply:
Done...